### ISSUE TITLE
SetDate() throws exception

### ISSUE THREAD

**Issue by mustafashujaie** [babakhani/pwt.datepicker#131]
I am using it in the following way:

```
<input type="text" class="form-control input-sm" id="j-start-date" />
<input type="text" class="form-control input-sm" id="j-end-date" />
```

```
$(document).ready(function () {
	$('#j-start-date')
		.pDatepicker({ format: 'YYYY-MM-DD' })
		.pDatepicker("setDate", [1392, 12, 1]);
	$('#j-end-date')
		.pDatepicker({ format: 'YYYY-MM-DD' })
		.pDatepicker("setDate", [1392, 12, 7]);
	});
});
```

The first call to `setDate` runs without any errors as expected. But the second call throws the following error:

```
Uncaught TypeError: this.timePicker.setTime is not a function
    at f.setTime (persian-datepicker-0.4.5.min.js:6)
    at f.setDate (persian-datepicker-0.4.5.min.js:6)
    at HTMLInputElement.<anonymous> (persian-datepicker-0.4.5.min.js:6)
    at Function.each (jquery-1.10.2.js:671)
    at init.each (jquery-1.10.2.js:280)
    at init.e.fn.persianDatepicker.e.fn.pDatepicker (persian-datepicker-0.4.5.min.js:6)
    at HTMLDocument.<anonymous> ((index):215)
    at fire (jquery-1.10.2.js:3062)
    at Object.fireWith [as resolveWith] (jquery-1.10.2.js:3174)
    at Function.ready (jquery-1.10.2.js:447)
    at HTMLDocument.completed (jquery-1.10.2.js:118)
```

**Comment by muhshahabipour** [comment]
Currently you must use "Develop" branch until release 0.5.0

https://github.com/babakhani/pwt.datepicker/tree/Develop

for npm is : npm install  git://github.com/babakhani/pwt.datepicker.git#Develop --save

**Comment by babakhani** [comment]
@muhshahabipour thanks for answer.
@mustafashujaie this issue fixed on new 0.5.5 version. please check this out and if problem still exist reopen this issue.
